test/mokernels/moinput.jl

```diff
@@ -44,4 +44,35 @@
         @test ibf[7] == (x[3], 1)
         @test all([(x_, i) for x_ in x for i in 1:3] .== ibf)
     end
+
+    @testset "prepare_isotopic_multi_output_data" begin
+        @testset "ColVecs" begin
+            N = 10
+            P = 5
```

Comment:
It doesn't matter much here, but in general I like to use prime numbers for array dimensions because that makes it a lot more obvious what's going on when something fails.
```suggestion
            N = 5
            P = 3
```